velox/exec/GroupingSet.h

```diff
@@ -43,6 +43,13 @@ class GroupingSet {
 
   ~GroupingSet();
 
+  // GroupingSet is also used for MarkDistinctOperator. When being used so,
+  // most aggregation related arguments are empty as they are not needed by
+  // groupingSet for our DISTINCT use case.
+  static std::unique_ptr<GroupingSet> createForMarkDistinct(
```

Comment:
Does MarkDistinct operator support spilling? If not, should it?

Reply:
That's a good question. I think MarkDistinct should support spilling.

Reply:
Would it be acceptable to add spilling in a separate PR? Assuming you don't feel as-is this PR would be in a "half-baked" state.

Reply:
@shiyu-bytedance This would be fine. Just mention that this version doesn't include spilling.

Reply:
@mbasmanova updated PR description to reflect this.